mood-analysis.rb

```diff
@@ -19,7 +19,57 @@ def analyze_mood(words)
   return ":-|"
 end
 
+def strip_punctuation(string)
+  string.gsub!(/[^0-9A-Za-z]/, ' ')
+end
+
+def happy_days(list_of_moods)
+  # how many tweets until there have been 3 happy days?
+  number_of_happy_days = 0
+  moods_read = 0
+  list_of_moods.each do |mood|
+    # THANK YOU RUBY KOANS!!!
+    break moods_read if number_of_happy_days == 3
```

Comment:
There's no reason to break here and then test the condition again later over just putting the return right here.